BoldGrid/weforms
# remove appsero

resolves #99 

### Testing

1. Install WordPress with weForms and Code Snippets
2. Add a Snippet with the following content (at any priority level): add_filter('run_wptexturize', '__return_false');
3. Create a post with double- and triple-dashes and preview it on the front-end.
4. Dashes should show as intended based on post and not be replaced with en- or em-dashes.
